### DISCUSSION TITLE
No audio when playing from a group inside a group

### DISCUSSION THREAD

**Discussion by JoeMBlair** [music-assistant/support#2783] [Q&A] [answered]
Whenever I try to play audio that's a Universal Group Player inside another Universal Group Player I get no audio. Music assistant still shows that the song is playing and the seek bar is still moving. I'm able to play audio when it's just inside one group.
I'm running Music Assistant 2.2.0b11

**Comment by OzGav** [comment]
I don’t know if this is supported. @marcelveldt ?

**Reply by marcelveldt** [reply, marked as answer]
No group inside group is not supported and actually a bug that it allowed you to create that, we have an open issue for that.
